packages/device-connector/src/deployment-server/service.ts

```diff
@@ -1,24 +1,12 @@
+import type { DeviceType, Device } from '@prisma/client'
```

Comment:
Just a Todo for the future, but we should share them through deployment server only